src/mechanics/tools/AdaptiveSolve.cpp

```diff
@@ -46,9 +46,9 @@ void AdaptiveSolve::Solve(double tEnd)
             if (t + dt > tEnd)
                 dt = tEnd - t;
         }
-        catch (NewtonRaphson::NoConvergence& e)
+        catch (...)
         {
```

Comment:
If something goes wrong in the postprocessing - well: decrease the timestep. ?

Reply:
It is a bit tricky here. The solver implementation may throw _some_ exception. Maybe its cause is _"SolverType MUMPS not found"_ and we have to abort. Maybe it is some _"Factorization error"_ where a decreased time step may help. But you are totally right that catching everything is bad. I'll try to think of a solution. Maybe catch outside of NewtonRaphson...